hapi/hedera-protobufs/block/stream/output/crypto_service.proto

```diff
@@ -62,7 +63,20 @@ message DeleteAllowanceOutput {}
  * This message SHALL NOT duplicate information already contained in
  * the original transaction.
  */
-message CreateAccountOutput {}
+message CreateAccountOutput {
+    /**
+     * A newly created account identifier.<br/>
+     * This is the account identifier for the account created as part of
+     * any transaction that creates a new account.<br/>
+     * This value is necessary due to the difficulty of otherwise identifying
+     * created accounts versus the other account updates necessary for every
+     * transaction (e.g. fee payment).
+     * <p>
+     * This value SHALL be set for every account creation.
+     * This value SHALL NOT be set if no account is created.
+     */
+    proto.AccountID created_account = 1;
```

Comment:
```suggestion
    proto.AccountID created_account_id = 1;
```

Reply:
Based on this PR https://github.com/hashgraph/hedera-services/pull/17663 It was suggested to fix here.

Reply:
Updated.